src/components/services/Services.tsx

```diff
@@ -37,6 +39,15 @@ export function Services(props: Props) {
   const account = useSelector(state => (state.core.account !== defaultAccount ? state.core.account : undefined))
   const { navigation } = props
 
+  // Methods to call once all of the currency wallets have been loaded
+  useAsyncEffect(async () => {
+    if (account == null) return
```

Comment:
On logout, account will be a dummy object, but the dummy object won't have a `waitForAllWallets` method. So maybe check `account?.waitForAllWallets != null` to avoid this case.